rustfmt-core/rustfmt-lib/tests/target/issue-3550.rs

```diff
@@ -0,0 +1,39 @@
+fn main() {
```

Comment:
for #3550 i think it would be good to have a test under `source` that has the original snippet (plus something that gets formatted to the other snippet added here) and then here showing that formatted as expected

Reply:
Done